### ISSUE TITLE
Transation and TransactionReceipt similarity

### ISSUE THREAD

**Issue by rockfridrich** [BlockStoreApp/Web3Swift#92]
`Transaction` and `TransactionReceipt` should be the same object. Because `TransactionReceipt` represents extended data of `Transaction` that was mine on the network.

**Comment by 0crat** [comment]
@biboran[/z](https://www.0crat.com/u/biboran) please, pay attention to this issue

**Comment by biboran** [comment]
@rockfridrich I agree to some extent; however, how do you see the interface of the `Transaction`? It obviously cannot contain all of the properties outlined in the JSON RPC response.
